README.md

```diff
@@ -1,6 +1,10 @@
 # Documentation
 
+Documentation for the [redacted]  project.
```

Comment:
```[redacted]``` ? Otherwise, Looks good to me 👍

Reply:
`[redacted]` is weird to me too.  Per @0x5c in slack. Happy to change it back if you (@Ne02ptzero) are cool with it.

Reply:
@0x5c ?